### DISCUSSION TITLE
right options for expression wrap

### DISCUSSION THREAD

**Discussion by ingDIY** [uncrustify/uncrustify#4330] [Q&A]
Hello,
I'm trying to configure wrapping for long expressions but I cant understand why this expression is not left ontouched.
![Immagine 2024-06-20 164436](https://github.com/uncrustify/uncrustify/assets/10012263/6beff8f6-3a76-4d3d-92cc-30ab10c25fb6)

What options should I check?
Thanks!

PS: i attach my current config file
[uncrustify_config.txt](https://github.com/user-attachments/files/15915766/uncrustify_config.txt)

**Comment by guy-maurel** [comment]
see the comment before
